ui/src/containers/BasicDetails/index.tsx

```diff
@@ -40,14 +40,9 @@ import './BasicDetails.css';
 
 const BasicDetails: React.FC = () => {
   const { resources, catalogs } = useMst();
-  const {
-    catalog,
-    kind,
-    name,
-    version
-  }: { catalog: string; kind: string; name: string; version: string } = useParams();
+  const { catalog, kind, name, version } = useParams();
```

Comment:
why is this changed?

Reply:
With react router dom update, this was giving a type error, hence added a change